app.js

```diff
@@ -46,8 +46,15 @@ app.get("/login", (req, res) => {
 // 3) logout route (no page)
 
 // 4) employee page
-app.get("/employee", (req, res) => {
-  res.render("pages/employee")
+app.get("/employee/:userId(\\d+)", (req, res) => { // To have more control over the exact string that can be matched by a route parameter, you can append a regular expression in parentheses (())
+  database.any(`SELECT * FROM schedules JOIN users ON users.id = schedules.user_id WHERE schedules.user_id = ${req.params.userId};`)
```

Comment:
You should pass parameters to the database as an argument, not as part of the query, as passing it this way could lead to SQL injections: https://en.wikipedia.org/wiki/SQL_injection.

```javascript
database.any("SELECT * FROM schedules JOIN users ON users.id = schedules.user_id WHERE schedules.user_id = $1;", req.params.userId)
```
(you might also need to convert req.params.userId from string to an integer, not sure)